rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -773,7 +778,7 @@ def load(
         cls,
         meta: Dict[Text, Any],
         model_dir: Text = None,
-        model_metadata: "Metadata" = None,
+        model_metadata: Metadata = None,
         cached_component: Optional["DIETClassifier"] = None,
```

Comment:
do we actually use it?

Reply:
no

Reply:
not sure where it actually comes from

Reply:
I think it comes from the fact that SpacyNLP can be cashed